### PR TITLE
Add "-o json" option for "minikube addon images"

### PR DESCRIPTION
**Pull request by thomasjm**
I wanted to be able to programmatically consume the output of `minikube addon images ADDON_NAME`.

The normal behavior is unchanged:

```shell
$ minikube addons images gvisor
    ▪ gvisor has the following images:
|-------------|----------------------------------------------------------------------------------------------------------|------------------|
| IMAGE NAME  |                                              DEFAULT IMAGE                                               | DEFAULT REGISTRY |
|-------------|----------------------------------------------------------------------------------------------------------|------------------|
| GvisorAddon | k8s-minikube/gvisor-addon:v0.0.1@sha256:d29adbddc7a44dab4c675c0eaea398907fe1f33f5f723274b0d40195b3076cae | gcr.io           |
|-------------|----------------------------------------------------------------------------------------------------------|------------------|
```

But now you can do `-o json`:

```shell
$ minikube addons images gvisor -o json
["gcr.io/k8s-minikube/gvisor-addon:v0.0.1@sha256:d29adbddc7a44dab4c675c0eaea398907fe1f33f5f723274b0d40195b3076cae"]
```

Also fixed the grammar of the existing message from "has following images" to "has the following images."

### CODE REVIEW

**Review by medyagh — CHANGES_REQUESTED** [review]
this seems like a great feature and thanks for adding it, just a couple comments

File: cmd/minikube/cmd/config/images.go [hunk 1]
```diff
@@ -38,33 +43,65 @@ var addonsImagesCmd = &cobra.Command{
...
+
+			for imageName, defaultImage := range conf.Images {
+				tData = append(tData, []string{imageName, defaultImage, conf.Registries[imageName]})
 			}
+
+			table.AppendBulk(tData)
+			table.Render()
 		} else {
-			out.FailureT("No such addon {{.name}}", out.V{"name": addon})
+			out.Infof("{{.name}} doesn't have images.", out.V{"name": addon})
```
[the remark this answers is not in the document]
Comment (thomasjm) [reply cmd/minikube/cmd/config/images.go:78]: The diff is being a little confusing here by splitting the old function in half -- there's no logic change here. The "previous" line can still be found below.

File: cmd/minikube/cmd/config/images.go [hunk 2]
```diff
@@ -38,33 +43,65 @@ var addonsImagesCmd = &cobra.Command{
...
 		}
-	},
+	} else {
+		out.FailureT("No such addon {{.name}}", out.V{"name": addon})
+	}
+}
+
+func printAddonImagesJson(addon string) {
```
[the remark this answers is not in the document]
Comment (thomasjm) [reply cmd/minikube/cmd/config/images.go:85]: Done

**Comment by medyagh** [comment]
thanks for this contribution, look forward to see more contributions

**Pull request merged**